spec/lib/listen/record_spec.rb

```diff
@@ -220,9 +243,17 @@ def record_tree(record)
         it { should eq('file.rb' => { mtime: 1.1 }) }
       end
 
-      context 'with path/subdir' do
+      context 'with empty path/subdir' do
         before { record.add_dir('path/subdir') }
-        it { should eq('subdir' => {}) }
```

Comment:
@bryanlira you added this test in https://github.com/guard/listen/pull/460. Do you remember why you expected it to return this, vs an empty hash? The [acceptance test](https://github.com/guard/listen/pull/460/files#diff-cb0c4f7e5adeaa9e4106f4a3f5a73261de4e95b52f0367868a3dd40a2e74b9efR198) you added in that PR is still passing with this branch, so I'm not sure if this unit test was critical for something else that I'm not seeing it.